cls/SourceControl/Git/Build.cls

```diff
@@ -0,0 +1,18 @@
+Class SourceControl.Git.Build
+{
+
+ClassMethod BuildUIForDevMode(devMode As %Boolean, rootDirectory As %String)
+{
+    if 'devMode {
+        return
+    }
+    write !, "In developer mode, building web UI:"
+    set webUIDirectory = ##class(%File).SubDirectoryName(rootDirectory, "git-webui")
+    write !, "npm install"
+    write !, $zf(-100, "/SHELL", "cd", webUIDirectory, "&&", "npm", "install")
```

Comment:
This should be npm install --prefix webUIDirectory rather than using cd. (Same below, npm run --prefix)

Reply:
At least for me, `npm install` does not respect `--prefix`. But `npm ci --prefix` does work.